apps/drawing-tools.html

```diff
@@ -586,20 +600,36 @@
       return Microsoft.Maps.SpatialMath.Geometry.bounds(shapes);
     }
 
+
+
     /**
      * Callback to initialize map and event handlers.
      *
      * @return {void}
      */
     function getMap() {
-      Microsoft.Maps.loadModule(["Microsoft.Maps.DrawingTools", "Microsoft.Maps.GeoJson", "Microsoft.Maps.SpatialMath", "Microsoft.Maps.WellKnownText"], function() {
+      Microsoft.Maps.loadModule(["Microsoft.Maps.AutoSuggest", "Microsoft.Maps.DrawingTools", "Microsoft.Maps.GeoJson", "Microsoft.Maps.SpatialMath", "Microsoft.Maps.WellKnownText"], function() {
         var map = new Microsoft.Maps.Map("#myMap", {
           disableBirdseye: true,
           disableStreetside: true,
           disableStreetsideAutoCoverage: true,
           showMapTypeSelector: true,
         });
 
+        var layer = new Microsoft.Maps.Layer();
```

Comment:
Please set the "zIndex" attribute of the new layer. Make sure to select a z-index that does not introduce a regression (with respect to the capabilities of the drawing manager's toolbar, e.g., editing an existing polygon).